packages/inventory/src/containers/ServicePointProfile/utils.tsx

```diff
@@ -44,7 +44,8 @@ export const getCords = (geoJson?: Partial<Geometry>) => {
     Array.isArray(geoJson.coordinates) &&
     geoJson.coordinates.length === 2
   ) {
-    return geoJson.coordinates.join(', ');
+    const coordinatesOfInterest = [...geoJson.coordinates].reverse();
+    return coordinatesOfInterest.join(', ');
```

Comment:
@HaseemUlHaq how do you feel about parsing the geoJson here to return an object containing 2 props one for the latitude and another for the longitude

Reply:
That can also be done, but this function is only meant to return comma separated lat long value

Reply:
that can be still composed at the point of use, no? i.e. we can still easily generate a comma separated string of the coordinates even if we return an object with the above 2 properties.